htdocs/xoops_lib/Xoops/Core/Events.php

```diff
@@ -167,7 +167,7 @@ protected function setEvents()
      * Trigger a specific event
      *
      * @param string $eventName Name of the event to trigger
-     * @param mixed  $args      Method arguments
+     * @param array  $args      array of Method arguments
```

Comment:
This is not correct. The single argument is not of any specified type. Some uses will be objects.